packages/product-components/src/components/SelectAssetModal/AssetItemNotFound.tsx

```diff
@@ -1,61 +1,27 @@
-import { FormattedMessage } from 'react-intl';
+import { ReactNode } from 'react';
 
 import { Column, Paragraph, Text } from '@trezor/components';
 import { spacings } from '@trezor/theme';
 
-import { SelectAssetNetworkProps, SelectAssetSearchCategoryType } from './SelectAssetModal';
-
 interface AssetItemNotFoundProps {
-    searchCategory: SelectAssetSearchCategoryType;
-    networkCategories: SelectAssetNetworkProps[];
+    noItemsAvailablePlaceholder: { heading: string | ReactNode; body?: string | ReactNode };
```

Comment:
`string` is assignable to `ReactNode`
```suggestion
    noItemsAvailablePlaceholder: { heading: ReactNode; body?: ReactNode };
```